thunderbird/processes/wps_split_merged_climos.py

```diff
@@ -0,0 +1,118 @@
+# Processor imports
+from pywps import (
+    Process,
+    LiteralInput,
+    ComplexInput,
+    ComplexOutput,
+    FORMATS,
+)
+from pywps.app.Common import Metadata
+
+# Tool imports
+from nchelpers import CFDataset
+from dp.split_merged_climos import split_merged_climos
+from thunderbird.utils import (
+    MAX_OCCURS,
+    is_opendap_url,
+    collect_output_files,
+    build_meta_link,
+)
+
+# Library imports
+import logging
+import os
+import math
+
+logger = logging.getLogger("PYWPS")
+
+
+class SplitMergedClimos(Process):
+    def __init__(self):
+        self.log_level_choices = [
+            "NOTSET",
+            "DEBUG",
+            "INFO",
+            "WARNING",
+            "ERROR",
+            "CRITICAL",
+        ]
+        inputs = [
+            ComplexInput(
+                "netcdf",
+                "NetCDF Datasets",
+                abstract="NetCDF files to process",
+                min_occurs=1,
+                max_occurs=MAX_OCCURS,
+                supported_formats=[FORMATS.NETCDF, FORMATS.DODS],
+            ),
+            LiteralInput(
+                "loglevel",
+                "Log Level",
+                default="INFO",
+                abstract="Logging level",
+                allowed_values=self.log_level_choices,
+            ),
+        ]
+        outputs = [
+            ComplexOutput(
+                "output",
+                "Output",
+                abstract="Collection of split climatological files",
+                as_reference=True,
+                supported_formats=[FORMATS.META4],
+            ),
+        ]
```

Comment:
I'm starting to think we might be able to factor out these inputs/outputs. For the most part the parameters are the same for each process and we could save on the instantiation code by importing pre-built io objects.  We could create a `utils` file called `wps_io.py` (or something along those lines) and define our inputs/outputs in there.  Then use them to build the io lists for each process.  What do you think?

Reply:
I think that could work.